src/PKSim.Infrastructure/ProjectConverter/v10/Converter9To10.cs

```diff
@@ -189,21 +192,24 @@ private void convertIndividualTransporter(IndividualMolecule moleculeToConvert,
          //Ensure we update the default direction based on the selected transporter type
          allTransporterExpressionContainers.Each(x=>x.TransportDirection = TransportDirections.DefaultDirectionFor(newTransporter.TransportType, x));
 
-         moleculeToConvert.GetAllChildren<TransporterExpressionContainer>().Each(x => convertTransportContainer(x, allTransporterExpressionContainers, newTransporter));
+         moleculeToConvert.GetAllChildren<TransporterExpressionContainer>().Each(x => convertTransportContainer(x, allTransporterExpressionContainers));
       }
 
-      private void convertIndividualProtein(IndividualMolecule moleculeToConvert, IndividualMolecule newMolecule)
+      private void convertIndividualProtein(IndividualMolecule moleculeToConvert, IndividualMolecule newMolecule, Individual individual)
       {
          var proteinToConvert = moleculeToConvert as IndividualProtein;
          var newProtein = newMolecule as IndividualProtein;
          if (proteinToConvert == null || newProtein == null)
             return;
 
          newProtein.Localization = proteinToConvert.Localization;
+         //Set set it first to none to ensure that it is set properly after reading from the snapshot file
+         newProtein.Localization = Localization.None;
+         _moleculeExpressionTask.SetExpressionLocalizationFor(newProtein, proteinToConvert.Localization, individual);
```

Comment:
@Yuri05 This was the line missing. Based on the localization, we need to update the expression